docs/pages/guides/local-first.mdx

```diff
@@ -38,6 +38,17 @@ A comprehensive list of tools is available on the ["Local-first software" commun
 
 One way to think of local-first tools is to group them by the following categories: persistence, state management, and syncing. Some tools will fit into multiple categories if they handle multiple aspects of the problem. Syncing can be further subdivided into syncable data structures and transport layers.
 
+### Legend-State
+
+[Legend-State](https://legendapp.com/open-source/state/v3/) is a super fast all-in-one state and sync library that lets you write less code to make faster apps. Legend-State has four primary goals:
+
+1. As easy as possible to use.
+1. The fastest React state library.
+1. Fine-grained reactivity for minimal renders.
+1. Powerful sync and persistence (with Supabase support built in!)
```

Comment:
```suggestion
- The fastest React state library.
- Fine-grained reactivity for minimal renders.
- Powerful sync and persistence (with Supabase support built-in)
```

We try to reserve numbered steps when describing a procedure. I think first point is a bit opinionated on whether someone finds a particular library easy to use, in general. Also don't think it adds anything relevant here since you've already mentioned that "lets you write less code to make faster apps" in the intro.

Let's also avoid using `!` since using brackets to convey the info about the built-in support has enough emphasis.